tests/e2e/workloads/amq/test_amq_streamer_creation.py

```diff
@@ -0,0 +1,60 @@
+import logging
+import pytest
+import time
+
+from ocs_ci.framework.testlib import E2ETest, workloads
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.amq import AMQ
+from tests.helpers import default_storage_class
+
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope='function')
+def test_fixture_amq(request):
+
+    amq = AMQ()
+
+    def teardown():
+        amq.cleanup()
+
+    request.addfinalizer(teardown)
+    return amq
+
+
+@workloads
+class TestAMQBasics(E2ETest):
+    @pytest.mark.parametrize(
+        argnames=["interface"],
+        argvalues=[
+            pytest.param(
+                constants.CEPHBLOCKPOOL, marks=pytest.mark.polarion_id("OCS-612")
+            ),
+            pytest.param(
+                constants.CEPHFILESYSTEM, marks=pytest.mark.polarion_id("OCS-612")
+            )
+        ]
+    )
+    def test_install_amq_cephfs(self, interface, test_fixture_amq):
+        """
+        Create amq cluster and run open messages on it
+
+        """
+        # Get sc
+        sc = default_storage_class(interface_type=interface)
+
+        # Deploy amq cluster
+        test_fixture_amq.setup_amq_cluster(sc.name)
+
+        # Run open messages
+        test_fixture_amq.create_messaging_on_amq()
```

Comment:
it would be great to test with multiple pods of producers and consumers -> may be around 5 minimum. does that work?